docs/content/getting-started/quick-start-with-kubernetes.md

```diff
@@ -0,0 +1,312 @@
+metadata:
+  name: traefik-role-binding
+
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: traefik-role
+subjects:
+  - kind: ServiceAccount
+    name: traefik-account
+    namespace: default # Using "default" because we didn't specify a namespace when creating the ClusterAccount.
+```
+
+!!! info
+    
+    `roleRef` is the kubernetes reference to the role created in `00-role.yml`.
+    
+    `subjects` is the list of accounts reference. In the guilde, it
+    only contains the account created in `00-account.yml`
+
+## Deployment and Exposition
+
+The [ingress controller](https://traefik.io/glossary/kubernetes-ingress-and-ingress-controller-101/#what-is-a-kubernetes-ingress-controller)
+is a software that runs in the same way as any other application on a cluster.
+In order to start Traefik on the Kubernetes cluster,
+a [`Deployment`](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/deployment-v1/) resource must exist to describe how to configure
+and scale containers horizontally to support larger workloads.
+
+Start by creating a file called `02-traefik.yml` and paste the following `Deployment` resource:
+
+```yaml tab="02-traefik.yml"
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: traefik-deployment
+  labels:
+    app: traefik
+
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: traefik
+  template:
+    metadata:
+      labels:
+        app: traefik
+    spec:
+      serviceAccountName: traefik-account
+      containers:
+        - name: traefik
+          image: traefik:v2.7
+          args:
+            - --api.insecure
+            - --providers.kubernetesingress
+          ports:
+            - name: web
+              containerPort: 80
+            - name: dashboard
+              containerPort: 8080
+```
+
+The deployment contains an important attribute for customizing Traefik: `args`.
+These arguments are the static configuration for Traefik.
+From here, it is possible to enable the dashboard, configure entry points,
+select dynamic configuration providers, and [more](../reference/static-configuration/cli.md)...
+
+In this deployment, the static configuration enables the Traefik dashboard and uses Kubernetes native
+ingress resources as router definitions to route incoming requests.
+
+!!! info
+    - If there is no entry points in the static configuration,
+      Traefik creates a default one called `web` using the port `80` routing HTTP requests.
+    - When enabling the `api.insecure` mode, Traefik exposes the dashboard on the port `8080`.
+
+A deployment manages scaling and then can create a tons of containers, called Pods.
+Each pod is configured following the `spec` field in the deployment.
+Given that it can exist a lot of Traefik instances, a piece is required to forward the traffic
+to any of the instance: a [`Service`](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/#Service).
```

Comment:
```suggestion
A deployment manages scaling and then can create lots of containers, called [Pods](https://kubernetes.io/docs/concepts/workloads/pods/).
Each pod is configured following the `spec` field in the deployment.
Given that it can exist a lot of Traefik instances,
a piece is required to forward the traffic to any of the instance:
a [`Service`](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/#Service).
```